README.md

```diff
@@ -27,19 +53,20 @@ The data model/DB schema lives in `j-db-client/prisma/schema`, server side code
    $ git clone git@github.com:Journaly/journaly.git
    ```
 1. Locate the `.env.example` file and copy the contents into a new `.env` file alongside it.
+1. Navigate to `packages/web`
 1. Run `npm ci`
-1. Run `npm run prisma:generate`
```

Comment:
@Lanny removing this step because it should happen with the prisma `post-install` hook, right?

Reply:
In theory yes, I think we needed to add it to the `build` npm script because circle was restoring node_modules from a cache and not re-running it? Actually that brings up a good point that you need to run `npm run build` at least once for nexus reflection and all the static analysis stuff that kids love these days to work, so that would probably be good to add here